src/main/java/net/dries007/tfc/objects/te/TECropBase.java

```diff
@@ -0,0 +1,69 @@
+package net.dries007.tfc.objects.te;
+
+import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import net.minecraft.nbt.NBTTagCompound;
+
+import net.dries007.tfc.objects.blocks.agriculture.BlockCropTFC;
+import net.dries007.tfc.util.calendar.CalendarTFC;
+import net.dries007.tfc.util.calendar.ICalendarTickable;
+
+@ParametersAreNonnullByDefault
+public class TECropBase extends TETickCounter implements ICalendarTickable
+{
+    // static constants
+
+    private static final String NBT_LAST_TICK_CAL_CHECKED = "lastTickCalChecked";
+
+    // fields
+
+    protected long _lastTickCalChecked;
```

Comment:
```suggestion
    protected long lastTickCalChecked;
```

Reply:
I've removed all the underscores for fields.